_sources/Strings/Exercises.rst

```diff
@@ -75,17 +75,28 @@ Exercises
 
            .. actex:: ex_8_3
 
-              Assign to a variable in your program a triple-quoted string that contains
-              your favorite paragraph of text - perhaps a poem, a speech, instructions
-              to bake a cake, some inspirational verses, etc.
+                Assign to a variable in your program a triple-quoted string that contains
+                your favorite paragraph of text - perhaps a poem, a speech, instructions
+                to bake a cake, some inspirational verses, etc.
    
-              Write a function that counts the number of alphabetic characters (a through z, or A through Z) in your text and then keeps track of how many are the letter 'e'.  Your function should print an analysis of the text like this::
+                Write a function that counts the number of alphabetic characters (a through z, or A through Z) in your text and then keeps track of how many are the letter 'e'.  Your function should print an analysis of the text like this::
    
-                  Your text contains 243 alphabetic characters, of which 109 (44.8%) are 'e'.
-              ~~~~
-              def count(p):
-                  # your code here
+                    Your text contains 243 alphabetic characters, of which 109 (44.8%) are 'e'.
+                ~~~~
+                def count(p):
+                    # your code here
+
+                ====
+                from unittest.gui import TestCaseGui
+
+                class myTests(TestCaseGui):
+                    def testOne(self):
+                        string1 = "e"
+                        self.assertEqual(count(string1), 1, "Just one e")
```

Comment:
I would like to see more tests added here. A mixed string for example an empty string, a long string with no e's are some ideas.

Reply:
This initially had more tests but I think this was the pull request that had some merge conflicts, I will attempt to revert those changes and submit a new pull request.